deploy/k8s/charts/trustification/values.schema.json

```diff
@@ -804,6 +804,23 @@
             }
           ]
         },
+        "report": {
+          "description": "Configuration options for report.\n",
+          "allOf": [
+            {
+              "$ref": "#/definitions/Feature"
+            },
+            {
+              "$ref": "#/definitions/Image"
+            },
+            {
```

Comment:
Does this module really support metrics and tracing?

Reply:
I saw metrics and tracing are empty. so let is empty in here . so do I need to add it, let they will enable in next-step?

Reply:
I don't think there's support for infrastructure stuff in this case. So it should be removed consistently, like for the spog-ui nginx.

Reply:
ok